lib/rules/prefer-pascal-case.ts

```diff
@@ -87,8 +81,10 @@ export = createStorybookRule({
                 const scope = context.getScope().childScopes[0]
                 if (scope) {
                   const variable = ASTUtils.findVariable(scope, name)
-                  for (let i = 0; i < variable?.references?.length; i++) {
-                    const ref = variable.references[i]
+                  const index = variable?.references?.length || 0
```

Comment:
maybe `index` should be called something else? like `referenceCount`? in the next line `i` refers to index so it might be slightly confusing

Reply:
As far as I remember, it was not used at all. I am unsure whether it is a relic of the past or a draft of a future feature. Of course, I can add it back.

Reply:
Sounds good to me!